src/battle_game/core/game_core.h

```diff
@@ -117,6 +118,7 @@ class GameCore {
   void SetRenderPerspective(uint32_t player_id);
   [[nodiscard]] uint32_t GetRenderPerspective() const;
   [[nodiscard]] glm::vec4 GetPlayerColor(uint32_t player_id) const;
+  [[nodiscard]] glm::vec4 GetPlayerColorbooboo(uint32_t player_id) const;
```

Comment:
What is this for? If this is not generally used, pls do this in your local codes. you can use GetRenderPerspective() to get the internal value render_perspective_